deployment/localup/localup.sh

```diff
@@ -152,9 +149,11 @@ function make_config() {
     sed -i -e "s/Dsn = '.*'/Dsn = \"${USER}:${PWD}@tcp(${ADDRESS})\/${DATABASE}?parseTime=true\&multiStatements=true\&loc=Local\&interpolateParams=true\"/g" config.toml
 
     # manager
+    sed -i -e "s/SubscribeSPExitEventIntervalSec = .*/SubscribeSPExitEventIntervalSec = 100/g" config.toml
+    sed -i -e "s/SubscribeSwapOutExitEventIntervalSec = .*/SubscribeSwapOutExitEventIntervalSec = 100/g" config.toml
+    sed -i -e "s/SubscribeBucketMigrateEventIntervalSec = .*/SubscribeBucketMigrateEventIntervalSec = 100/g" config.toml
```

Comment:
Refine to `SubscribeBucketMigrateEventIntervalMillisecondSec`.

Reply:
fixed